src/sortStudents.ts

```diff
@@ -1,16 +1,53 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function calculateAverage(numbers: number[]): number {
+  return numbers.reduce((acc, curr) => acc + curr, 0) / numbers.length;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: string,
+): Student[] {
+  return [...students].sort((a, b) => {
+    switch (sortBy) {
+      case 'name':
+      case 'surname':
```

Comment:
and so on.
Basically that's the purpose for using enums - they are far more reliable then just the string values
```suggestion
      case SortBy.Name:
      case SortBy.Surname:
```